### PR TITLE
[JENKINS-64341] Fix for tables-to-divs

### PR DESCRIPTION
**Pull request by dariver**
Jira: [JENKINS-64341](https://issues.jenkins-ci.org/browse/JENKINS-64341)

### What has been done
1. Fix for tables-to-div regression

### Screenshots

Before | After
:-: | :-:
before-job|after-job|
before-global-config|after-global-config|

### How to test
1. Check configuration in `Manage Jenkins > Configure system`
2. Create a Freestyle job, add a new build step action and fill the values
3. Save and verify that all the values were persisted.

### CODE REVIEW

**Comment by dariver** [comment]
cc @timja, @fqueiruga, @fbelzunc

File: src/main/resources/jenkins/plugins/publish_over_ftp/BapFtpPublisher/config.jelly [hunk 1]
```diff
@@ -66,14 +66,14 @@
         <j:scope>
             <j:set var="descriptor" value="${descriptor.transferDescriptor}"/>
             <f:repeatable items="${instance.transfers}" var="instance" name="transfers" minimum="1" header="${m.transfers_dragAndDropLabel()}" add="${m.addTransfer()}">
-                <table width="100%" padding="0">
+                <p:blockWrapper>
```
Comment (timja) [inline src/main/resources/jenkins/plugins/publish_over_ftp/BapFtpPublisher/config.jelly:69]: isn't there more files that need changing? see https://github.com/jenkinsci/publish-over-ftp-plugin/search?q=table

Comment (dariver) [reply src/main/resources/jenkins/plugins/publish_over_ftp/BapFtpPublisher/config.jelly:69]: Ups! You are totally right

**Review by timja — APPROVED** [review]

**Comment by dariver** [comment]
It seems that this PR contains the same changes in https://github.com/jenkinsci/publish-over-ftp-plugin/pull/8.

Feel free to close this and merge the other.

**Review by fqueiruga — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Got permissions in https://groups.google.com/u/1/g/jenkinsci-dev/c/XXztPiLDPmM

**Pull request merged**